web/crux/prisma/schema.prisma

```diff
@@ -371,17 +373,25 @@ enum AuditLogRequestMethodEnum {
   delete
 }
 
-model AuditLog {
-  id        String                     @id @default(uuid()) @db.Uuid
-  createdAt DateTime                   @default(now()) @db.Timestamptz(6)
-  userId    String                     @db.Uuid
-  teamId    String                     @db.Uuid
-  context   AuditLogContextEnum
-  method    AuditLogRequestMethodEnum?
-  event     String
-  data      Json?
+enum AuditLogActorTypeEnum {
+  user
+  deploymentToken
+}
 
-  team Team @relation(fields: [teamId], references: [id], onDelete: Cascade)
+model AuditLog {
+  id                String                     @id @default(uuid()) @db.Uuid
+  createdAt         DateTime                   @default(now()) @db.Timestamptz(6)
+  actorType         AuditLogActorTypeEnum
+  userId            String?                    @db.Uuid
```

Comment:
Using only one column to store actorId and use `actorId` as a name.

Reply:
It is necessary because of the foreign key. :(